core/common/src/rpc/web/RpcMultipart.ts

```diff
@@ -48,12 +48,8 @@ export class RpcMultipart {
     form.append("objects", value.objects);
 
     for (let i = 0; i !== value.data.length; ++i) {
-      if (typeof (Blob) !== "undefined") {
-        form.append(`data-${i}`, new Blob([value.data[i]], { type: "application/octet-stream" }));
-      } else {
-        const buf = value.data[i];
-        form.append(`data-${i}`, Buffer.from(buf.buffer, buf.byteOffset, buf.byteLength));
-      }
+      const buf = value.data[i];
```

Comment:
@swbsi mind taking a look at this change?

Reply:
will this change with https://github.com/iTwin/itwinjs-core/pull/4589? If so, what order should these prs complete in?
This then #4589?

Reply:
In #4589, I move the backend clause of this to the backend package.

I wasn’t planning to change the check for Blob. Does node 18 now have a Blob type?

Reply:
Yes - the root of this is Node 18 added a global Blob type with the added global Fetch

Reply:
I can reverse the logic on my decoupling PR then...and make this problem go away.

Reply:
We won’t check for Blob anymore...

Reply:
Got it - that seems like the cleanest approach, though it does push this PR back to 4.0, no?

Reply:
No, I’ll change it in #4859, the decoupling one that @aruniverse mentioned.

Reply:
resolving as the other PR has been merged.